app/views/finders/show.html.erb

```diff
@@ -109,20 +109,21 @@
               </div>
             <% end %>
           </div>
-          <%#= render_mustache('result_facet_tags', @results.to_hash) %>
-          <%= render "facet_tags", @results.to_hash %>
+          <div id="js-facet-tag-wrapper">
```

Comment:
Why was `js-facet-tag-wrapper` removed from the `facet_tags` partial and moved here? Do we need to render the `facet_tags` partial without it being wrapped in `js-facet-tag-wrapper`?

Reply:
It was moved from the partial because we're now rendering the partial in the backend and passing the resulting markup to the frontend via AJAX (previously done with mustache). The JS replaces the content of `js-facet-tag-wrapper` with this chunk, so if it was still inside the partial it would include itself. So after you'd changed the facets a few times, the markup would look like this:

```
<div id="js-facet-tag-wrapper">
   <div id="js-facet-tag-wrapper">
      <div id="js-facet-tag-wrapper">
         (content)
      </div>
   </div>
</div>
```

Reply:
Could you add this to the commit message?

Reply:
Added.